src/extension/inspire/src/main/java/org/geoserver/inspire/web/UniqueResourceIdentifiersEditor.java

```diff
@@ -152,24 +152,6 @@ protected void onError(AjaxRequestTarget target, Form<?> form) {
         };
         add(button);
         
-        // grab a seat... the way I'm adding this validator in onBeforeRender will be hard
-        // to stomach... however, could not find other way to add a validation to an editabl table, grrr
-        add(new IValidator<UniqueResourceIdentifiers>() {
-
-            @Override
-            public void validate(IValidatable<UniqueResourceIdentifiers> validatable) {
-                UniqueResourceIdentifiers identifiers = identifiersModel.getObject();
-                if(identifiers.size() == 0) {
-                  ValidationError error = new ValidationError();
-                  String message = new ParamResourceModel(
-                          "noSpatialDatasetIdentifiers", UniqueResourceIdentifiersEditor.this).getString();
-                  error.setMessage(message);
-                  validatable.error(error);
-                }
-                
-            }
-            
-        });
```

Comment:
There is no way to show a warning when pressing submit, the button either exits the page (and thus no warning will be shown) or does not.
What one would do is to add a warning on the side of the inspire components, and use ajax listeners to show/hide it every time a inspire component gets edited.